src/System.Buffers/src/System/Buffers/ArrayPoolEventSource.cs

```diff
@@ -22,7 +21,7 @@ internal enum BufferAllocationReason : int
             PoolExhausted
         }
```

Comment:
Thanks, Vance.  Regarding localization, yes, we do have localized resources, named appropriately, etc.  Based on your recommendation, though, I'll remove all such things.